ruapu.h

```diff
@@ -371,6 +378,15 @@ void ruapu_init()
         g_ruapu_isa_map[i].capable = ruapu_detect_isa(g_ruapu_isa_map[i].inst);
     }
 #else
+#if defined __openrisc__
```

Comment:
wrap these code into a static function, call it from here

Reply:
done